src/Services/privateApiService.js

```diff
@@ -1,68 +1,78 @@
-import axios from 'axios';
-const URL = 'http://ongapi.alkemy.org/private/api';
+import axios from "axios";
+const URL = "http://ongapi.alkemy.org/private/api";
 
 const config = {
-    headers: {
-        Group: 91                //Aqui va el ID del equipo!!
-    }
-}
+  headers: {
+    Group: 91, //Aqui va el ID del equipo!!
+  },
+};
 
 const axiosInstance = axios.create({
-    baseURL: 'http://ongapi.alkemy.org/private/api',
+  baseURL: "http://ongapi.alkemy.org/private/api",
 });
 
-axiosInstance.interceptors.request.use((config)=> {
-    config.headers.Authorization = getAuthorizationHeader();
-    config.headers.Group = 91;
-    return config
-})
+axiosInstance.interceptors.request.use((config) => {
+  config.headers.Authorization = getAuthorizationHeader();
+  config.headers.Group = 91;
+  return config;
+});
 
-const privateGet = async (url,id,params={}) => {
-    const idPlaceholder = id ? `/${id}` : '';
-    const { data } = axiosInstance.get(`${url}${idPlaceholder}`, {params})
-    return data
+const privateGet = async (url, id, params = {}) => {
+  const idPlaceholder = id ? `/${id}` : "";
+  const { data } = axiosInstance.get(`${url}${idPlaceholder}`, { params });
+  return data;
 };
 
 const Get = () => {
-    axios.get('https://jsonplaceholder.typicode.com/users', config)
-    .then(res => console.log(res))
-    .catch(err => console.log(err))
+  axios
+    .get("https://jsonplaceholder.typicode.com/users", config)
+    .then((res) => console.log(res))
+    .catch((err) => console.log(err));
 };
 
 const getAuthorizationHeader = () => {
-    const token = localStorage.getItem('token');
-    if(!token) return;
-    return {Authorization: `Bearer: ${token}`};
+  const token = localStorage.getItem("token");
+  if (!token) return;
+  return { Authorization: `Bearer: ${token}` };
 };
 
-const verifyProps = (path,id,body) => {
-    if (!path || !id || !body) {
-        throw new Error(`props not specified for async action put`);
-    };
+const verifyProps = (path, id, body) => {
+  if (!path || !id || !body) {
+    throw new Error(`props not specified for async action put`);
+  }
 };
+const privatePut = async (url, id, params) => {
+  const { data } = await axiosInstance.put(`${url}${id}`, params);
+  return data;
+};
```

Comment:
In my previous answer a few days ago I told you that I did not understand why it is not private. The method is calling axiosInstance who intercepts the request and checks the token. Anyway I delete it because Andrea's is already merged. Let that remain.